### DISCUSSION TITLE
FoxEss Wallbox

### DISCUSSION THREAD

**Discussion by FranzsGithub** [community/community#126473] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hallo zusammen, bezugnehmend auf diesen Hinweis bei (https://evcc.io/): "Ist dein Gerät noch nicht unterstützt, dann lass es uns auf [GitHub](https://github.com/evcc-io/evcc) wissen. Wenn es eine Schnittstelle hat, können wir es anbinden", frage ich an, wann die FoxEss Wallbox bei evcc unter "Geräte" auswählbar ist? Es hat eine RS485-Schnittstelle. Danke für eine Rückmeldung

**Comment by ebndev** [comment]
Hi @FranzsGithub, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
